MSAL/src/public/MSALResult.h

```diff
@@ -100,4 +100,14 @@
  */
 @property (readonly, nonnull) NSUUID *correlationId;
 
+/**
+ The authorization header for the specific authentication scheme . For instance "Bearer ..." or "Pop ...".
```

Comment:
Is there any RFC we can reference here for additional context?

Reply:
Good point. Added rfc reference.